mixer/adapter/stackdriver/config/config.proto

```diff
@@ -40,223 +40,177 @@ option (gogoproto.gostring_all) = false;
-                double growth_factor = 2;
-
-                // Must be greater than 0.
-                double scale = 3;
-            }
-
-            // Specifies a set of buckets with arbitrary widths.
-            //
-            // There are `size(bounds) + 1` (= `N`) buckets. Bucket `i` has the following
-            // boundaries:
-            //
-            // * Upper bound (`0 <= i < N-1`): `bounds[i]`
-            // * Lower bound (`1 <= i < N`): `bounds[i - 1]`
-            //
-            // The `bounds` field must contain at least one element. If `bounds` has
-            // only one element, then there are no finite buckets, and that single
-            // element is the common boundary of the overflow and underflow buckets.
-            message Explicit {
-                // The values must be monotonically increasing.
-                repeated double bounds = 1;
-            }
-
-            // Exactly one of these three fields must be set.
-            oneof definition {
-                // The linear buckets.
-                Linear linear_buckets = 1;
-
-                // The exponential buckets.
-                Exponential exponential_buckets = 2;
-
-                // The explicit buckets.
-                Explicit explicit_buckets = 3;
-            }
-        }
-
-        // For metrics with a metric value of DISTRIBUTION, this provides a mechanism
-        // for configuring the buckets that will be used to store the aggregated values.
-        // This field must be provided for metrics declared to be of type DISTRIBUTION.
-        // This field will be ignored for non-distribution metric kinds.
-        BucketsDefinition buckets = 3;
-
-        // Stackdriver metric type name, e.g.
-        // istio.io/service/server/request_count. If this is not provided, a
-        // concantenation of custom metric prefix (custom.googleapis.com/) and
-        // Istio metric name will be used.
-        string metric_type = 4;
+  // GCP Project to attach telemetry to. If a piece of data has a project ID
+  // or a MonitoredResource attached, that will take preference over this
+  // field.
+  string project_id = 2;
+
+  // Describes how to represent an Istio metric in Stackdriver.
+  // See
+  // https://github.com/googleapis/googleapis/blob/master/google/api/metric.proto
+  message MetricInfo {
+    // The kind of measurement for a metric, which describes how the data is
+    // reported. Ex: Gauge.
+    google.api.MetricDescriptor.MetricKind kind = 1;
+    // The type of the metric's value. Ex: Distribution.
+    google.api.MetricDescriptor.ValueType value = 2;
+
+    // For metrics with a metric value of DISTRIBUTION, this provides a
+    // mechanism for configuring the buckets that will be used to store the
+    // aggregated values. This field must be provided for metrics declared to
+    // be of type DISTRIBUTION. This field will be ignored for
+    // non-distribution metric kinds.
+    google.api.Distribution buckets = 3;
```

Comment:
Isn't this a breaking change for existing configuration?  Should we have a `google.api.Distribution distribution_buckets = 5` ?